test/tests/functions/image_function/image_resample.i

```diff
@@ -0,0 +1,44 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 20
+  ny = 20
+  nz = 40
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Functions]
+  [./image_func]
+    type = ImageFunction
+    file_base = stack/test
+    file_suffix = png
+    output_spacing = '1 1 0.5'
```

Comment:
Thanks Mike, now that I am playing with it again I am not sure what is doing. I will figure it out and add some proper documentation.